paymentsheet/src/main/java/com/stripe/android/paymentelement/embedded/DefaultEmbeddedActivityLauncher.kt

```diff
@@ -0,0 +1,42 @@
+package com.stripe.android.paymentelement.embedded
+
+import androidx.activity.result.ActivityResultCaller
+import androidx.activity.result.ActivityResultLauncher
+import androidx.lifecycle.DefaultLifecycleObserver
+import androidx.lifecycle.LifecycleOwner
+import com.stripe.android.lpmfoundations.paymentmethod.PaymentMethodMetadata
+
+internal interface EmbeddedActivityLauncher {
+    var formLauncher: ((code: String, paymentMethodMetadata: PaymentMethodMetadata?) -> Unit)?
```

Comment:
```suggestion
    val formLauncher: ((code: String, paymentMethodMetadata: PaymentMethodMetadata?) -> Unit)?
```

We don't want anyone other than the implementation to be able to change this value, so we should expose it as read only.

Reply:
Updated and made implementation private set